Mathlib/CategoryTheory/Bicategory/Functor/LocallyDiscrete.lean

```diff
@@ -0,0 +1,154 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Bicategory.Functor.Pseudofunctor
+import Mathlib.CategoryTheory.Bicategory.LocallyDiscrete
+
+/-!
+# Constructor for pseudofunctors from a locally discrete bicategory
+
+In this file, we define a constructor
+`pseudofunctorOfIsLocallyDiscrete` for the type `Pseudofunctor B C`
+when `C` is any bicategory, and `B` is a locally discrete category.
+Indeed, in this situation, we do not need to care about the field `map₂`
+of pseudofunctors because all the `2`-morphisms in `B` are identities.
+
+We also define a specialized constructor `LocallyDiscrete.mkPseudofunctor` when
+the source bicategory is of the form `B := LocallyDiscrete B₀` for a category `B₀`.
+
+A functor `F : I ⥤ B` with `B` a strict bicategory can also be promoted to a pseudofunctor
+(`Functor.toPseudofunctor`).
```

Comment:
```suggestion
# Pseudofunctors from locally discrete bicategories

This file provides various ways of constructing pseudofunctors from locally discrete
bicategories.

Firstly, we define the constructors `pseudofunctorOfIsLocallyDiscrete` and
`oplaxFunctorOfIsLocallyDiscrete` for defining pseudofunctors and oplax functors
from a locally discrete bicategories. In this situation, we do not need to care about
the field `map₂`,  because all the `2`-morphisms in `B` are identities.

We also define a specialized constructor `LocallyDiscrete.mkPseudofunctor` when
the source bicategory is of the form `B := LocallyDiscrete B₀` for a category `B₀`.

We also prove that a functor `F : I ⥤ B` with `B` a strict bicategory can be promoted
to a pseudofunctor (or oplax functor) (`Functor.toPseudofunctor`) with domain
`LocallyDiscrete I`.
```
I reworded this if you don't mind.